### PR TITLE
Added dire warning about docker run -rm

### PR DESCRIPTION
**Pull request by stuclem**
Fixes https://github.com/vmware/vic-product/issues/1463.

@matthewavery is this warning dire enough? Thanks!

### CODE REVIEW

File: docs/user_doc/vic_app_dev/using_volumes_with_vic.md [hunk 1]
```diff
@@ -177,4 +177,6 @@ volume rm <i>volume_name</i></pre>
 
 To remove a container and anonymous volumes joined to that container, run `docker rm -v`. If an anonymous volume is in use by another container, it is not removed.
 
-<pre>$ docker rm -v container1</pre>
\ No newline at end of file
+<pre>$ docker rm -v container1</pre>
```
Comment (matthewavery) [inline docs/user_doc/vic_app_dev/using_volumes_with_vic.md]: docker run --rm(?)

Comment (matthewavery) [reply docs/user_doc/vic_app_dev/using_volumes_with_vic.md]: Never mind, I misunderstood what this line was doing!

**Comment by matthewavery** [comment]
🐑 🇮🇹

File: docs/user_doc/vic_app_dev/using_volumes_with_vic.md (hunk 1 above)
Comment (stuclem) [reply docs/user_doc/vic_app_dev/using_volumes_with_vic.md]: Moved the comment about `docker run --rm` into a new section in https://github.com/vmware/vic-product/pull/1472/commits/72855cf803fcd38d21fabf57943da9b5f752f5c4

**Pull request merged**